app/views/smart_proxies/plugins/_dhcp_subnets.html.erb

```diff
@@ -0,0 +1,69 @@
+<ul id="proxy-dhcp-tab" class="nav nav-tabs nav-tabs-pf">
+  <li class="active"><a href="#dhcp_general"><%= _('General') %></a></li>
+  <% subnets.each do |subnet| %>
+    <li><a href="#<%= subnet.network.parameterize %>"><%= subnet_label subnet %></a></li>
+  <% end %>
+</ul>
+<div id="proxy-dhcp-tab-content" class="tab-content">
+  <div class="tab-pane active in" id="dhcp_general">
+    <div class="row">
+      <%= show_feature_version('dhcp') %>
```

Comment:
Yeah, this happens because `showProxies()` is not called when this page loads (run it manually from console and it will work...)